### PR TITLE
vegman: add host PCIe bifurcation configuration

### PR DESCRIPTION
**Pull request by alatarum**
Add CLI wrapper for host PCIe bifurcation configuration tool.

Signed-off-by: Andrei Kartashev <a.kartashev@yadro.com>

### CODE REVIEW

File: commands/host.vegman [hunk 1]
```diff
@@ -236,3 +236,21 @@ If connected via SSH, add extra ~ characters, one per SSH connection.
...
+ # Host configuration
+function cmd_config {
+  subcommand "$@"
+}
+
+# @sudo cmd_config_pcie admin
+# @restrict cmd_config_pcie admin
+# @doc cmd_config_pcie
+# Configure PCIe bifurcation
+#  -y, --yes  - does not ask for interactive confirmation
```
Comment (artemsen) [inline commands/host.vegman]: Copy/paste?

Comment (alatarum) [reply commands/host.vegman]: mmm... yes )

File: commands/host.vegman [hunk 2]
```diff
@@ -236,3 +236,21 @@ If connected via SSH, add extra ~ characters, one per SSH connection.
...
+ # Host configuration
+function cmd_config {
+  subcommand "$@"
+}
+
+# @sudo cmd_config_pcie admin
+# @restrict cmd_config_pcie admin
+# @doc cmd_config_pcie
+# Configure PCIe bifurcation
+#  Use -h or --help to show usage
```
Comment (artemsen) [inline commands/host.vegman]: We don't need this, all arguments (include -h) will be passed to host-pcie-cfg.

Comment (alatarum) [reply commands/host.vegman]: ok

**Review by nest1ing — APPROVED** [review]

**Review by artemsen — APPROVED** [review]

File: commands/host.vegman [hunk 3]
```diff
@@ -301,3 +301,20 @@ If connected via SSH, add extra ~ characters, one per SSH connection.
 EOF
   exec obmc-console-client
 }
+
+# @sudo cmd_config admin
+# @restrict cmd_config admin
+# @doc cmd_config
+ # Host configuration
+function cmd_config {
```
Comment (nest1ing) [inline commands/host.vegman:309]: Now it's a double (see line 231). Remove it.

**Review by nest1ing — CHANGES_REQUESTED** [review]

**Comment by alatarum** [comment]
We decide to not add tis to CLI

**Pull request closed without merging**